README.md
# NAME

Map::Metro::Plugin::Map::Brussels - Map::Metro map for Brussels

<div>
    <p>
    <img src="https://img.shields.io/badge/perl-5.16+-blue.svg" alt="Requires Perl 5.16+" />
    <a href="https://travis-ci.org/Csson/p5-Map-Metro-Brussels"><img src="https://api.travis-ci.org/Csson/p5-Map-Metro-Brussels.svg?branch=master" alt="Travis status" /></a>
    <a href="http://cpants.cpanauthors.org/release/CSSON/Map-Metro-Plugin-Map-Brussels-0.1002"><img src="http://badgedepot.code301.com/badge/kwalitee/CSSON/Map-Metro-Plugin-Map-Brussels/0.1002" alt="Distribution kwalitee" /></a>
    <a href="http://matrix.cpantesters.org/?dist=Map-Metro-Plugin-Map-Brussels%200.1002"><img src="http://badgedepot.code301.com/badge/cpantesters/Map-Metro-Plugin-Map-Brussels/0.1002" alt="CPAN Testers result" /></a>
    </p>
</div>

# VERSION

Version 0.1002, released 2019-04-30.

# SYNOPSIS

    use Map::Metro;
    my $graph = Map::Metro->new('Brussels')->parse;

Or:

    map-metro.pl route Brussels "Gare de l'Ouest" 'Centraal Station'

# DESCRIPTION

See [Map::Metro](https://metacpan.org/pod/Map::Metro) for usage information.

# Status

This map [contains](https://metacpan.org/pod/Map::Metro::Plugin::Map::Brussels::Lines):

- The four metro lines _L1_, _L2_, _L5_ and _L6_ \[[wikipedia](https://en.wikipedia.org/wiki/Brussels_Metro)\]
- The _T3_ tram line, but only between _Gare du Nord/Noordstation_ and _Albert_ \[[wikipedia](https://en.wikipedia.org/wiki/Brussels_tram_route_3)\]

# SOURCE

[https://github.com/Csson/p5-Map-Metro-Brussels](https://github.com/Csson/p5-Map-Metro-Brussels)

# HOMEPAGE

[https://metacpan.org/release/Map-Metro-Plugin-Map-Brussels](https://metacpan.org/release/Map-Metro-Plugin-Map-Brussels)

# AUTHOR

Erik Carlsson <info@code301.com>

# COPYRIGHT AND LICENSE

This software is copyright (c) 2016 by Erik Carlsson.

This is free software; you can redistribute it and/or modify it under
the same terms as the Perl 5 programming language system itself.


Changes
Revision history for Map-Metro-Plugin-Map-Brussels

{{$NEXT}}
 [API Changes]

 [Bug Fixes]

 [Documentation]

 [Enhancements]

 [New Features]

 [Other Changes]

 [Requirements]

 [Stats & Dependency Changes]

 [Testing]

0.1002 2019-04-30T19:09:33Z UTC
 [Other Changes]
 - Rebuilt with current dzil configuration.

 [Stats & Dependency Changes]
 - 4 files changed, 25 insertions(+), 24 deletions(-)
 - (run req) + Moose 2.1400
 - (run req) + namespace::autoclean 0.24
 - (run req) ~ Map::Metro 0.2206 --> 0.2300
 - (run req) ~ perl 5.014000 --> 5.016000
 - (run req) - Moose::Role
 - (test req) + ExtUtils::MakeMaker (any)
 - (test req) + File::Spec (any)
 - (test req) + IO::Handle (any)
 - (test req) + IPC::Open3 (any)
 - (test req) + Test::More 0.96
 - (test rec) + CPAN::Meta 2.120900
 - (conf req) + ExtUtils::MakeMaker (any)
 - (conf req) + File::ShareDir::Install 0.06
 - (conf req) - Module::Build::Tiny
 - (dev req) + Badge::Depot 0.0103
 - (dev req) + Badge::Depot::Plugin::Coverage (any)
 - (dev req) + Badge::Depot::Plugin::Cpantesters (any)
 - (dev req) + Badge::Depot::Plugin::Kwalitee (any)
 - (dev req) + Badge::Depot::Plugin::Perl (any)
 - (dev req) + Badge::Depot::Plugin::Travis (any)
 - (dev req) + Dist::Zilla 6.012
 - (dev req) + Dist::Zilla::Plugin::Authority 1.009
 - (dev req) + Dist::Zilla::Plugin::BumpVersionAfterRelease::Transitional
   0.008
 - (dev req) + Dist::Zilla::Plugin::ChangeStats::Dependencies::Git 0.0200
 - (dev req) + Dist::Zilla::Plugin::CheckChangesHasContent 0.008
 - (dev req) + Dist::Zilla::Plugin::Clean 0.07
 - (dev req) + Dist::Zilla::Plugin::ConfirmRelease (any)
 - (dev req) + Dist::Zilla::Plugin::CopyFilesFromBuild 0.170880
 - (dev req) + Dist::Zilla::Plugin::DistIller::MetaGeneratedBy (any)
 - (dev req) + Dist::Zilla::Plugin::ExecDir (any)
 - (dev req) + Dist::Zilla::Plugin::Git 2.045
 - (dev req) + Dist::Zilla::Plugin::Git::Check (any)
 - (dev req) + Dist::Zilla::Plugin::Git::CheckFor::CorrectBranch 0.014
 - (dev req) + Dist::Zilla::Plugin::Git::Commit (any)
 - (dev req) + Dist::Zilla::Plugin::Git::Contributors 0.034
 - (dev req) + Dist::Zilla::Plugin::Git::GatherDir (any)
 - (dev req) + Dist::Zilla::Plugin::Git::Push (any)
 - (dev req) + Dist::Zilla::Plugin::Git::Tag (any)
 - (dev req) + Dist::Zilla::Plugin::GithubMeta 0.54
 - (dev req) + Dist::Zilla::Plugin::InstallRelease 0.008
 - (dev req) + Dist::Zilla::Plugin::License (any)
 - (dev req) + Dist::Zilla::Plugin::MakeMaker (any)
 - (dev req) + Dist::Zilla::Plugin::Manifest (any)
 - (dev req) + Dist::Zilla::Plugin::ManifestSkip (any)
 - (dev req) + Dist::Zilla::Plugin::MapMetro::MakeGraphViz (any)
 - (dev req) + Dist::Zilla::Plugin::MapMetro::MakeLinePod (any)
 - (dev req) + Dist::Zilla::Plugin::MetaConfig (any)
 - (dev req) + Dist::Zilla::Plugin::MetaJSON (any)
 - (dev req) + Dist::Zilla::Plugin::MetaNoIndex (any)
 - (dev req) + Dist::Zilla::Plugin::MetaProvides::Package 2.004003
 - (dev req) + Dist::Zilla::Plugin::MetaYAML (any)
 - (dev req) + Dist::Zilla::Plugin::NextRelease::Grouped 0.0200
 - (dev req) + Dist::Zilla::Plugin::PodSyntaxTests (any)
 - (dev req) + Dist::Zilla::Plugin::PodWeaver 4.006
 - (dev req) + Dist::Zilla::Plugin::Prereqs (any)
 - (dev req) + Dist::Zilla::Plugin::PromptIfStale 0.055
 - (dev req) + Dist::Zilla::Plugin::Readme (any)
 - (dev req) + Dist::Zilla::Plugin::ReadmeAnyFromPod 0.163250
 - (dev req) + Dist::Zilla::Plugin::RewriteVersion::Transitional 0.007
 - (dev req) + Dist::Zilla::Plugin::RunExtraTests 0.028
 - (dev req) + Dist::Zilla::Plugin::ShareDir (any)
 - (dev req) + Dist::Zilla::Plugin::Test::Compile 2.057
 - (dev req) + Dist::Zilla::Plugin::Test::EOF 0.0501
 - (dev req) + Dist::Zilla::Plugin::Test::EOL 0.18
 - (dev req) + Dist::Zilla::Plugin::Test::Kwalitee::Extra v0.2.1
 - (dev req) + Dist::Zilla::Plugin::Test::NoTabs 0.15
 - (dev req) + Dist::Zilla::Plugin::Test::ReportPrereqs 0.027
 - (dev req) + Dist::Zilla::Plugin::Test::Version 1.05
 - (dev req) + Dist::Zilla::Plugin::TestRelease (any)
 - (dev req) + Dist::Zilla::Plugin::TravisYML (any)
 - (dev req) + Dist::Zilla::Plugin::UploadToCPAN (any)
 - (dev req) + Pod::Elemental::Transformer::List (any)
 - (dev req) + Pod::Weaver::Plugin::SingleEncoding (any)
 - (dev req) + Pod::Weaver::Plugin::Transformer (any)
 - (dev req) + Pod::Weaver::PluginBundle::CorePrep (any)
 - (dev req) + Pod::Weaver::Section::Authors (any)
 - (dev req) + Pod::Weaver::Section::Badges 0.0401
 - (dev req) + Pod::Weaver::Section::Collect (any)
 - (dev req) + Pod::Weaver::Section::GenerateSection 1.01
 - (dev req) + Pod::Weaver::Section::Generic (any)
 - (dev req) + Pod::Weaver::Section::Leftovers (any)
 - (dev req) + Pod::Weaver::Section::Legal (any)
 - (dev req) + Pod::Weaver::Section::Name (any)
 - (dev req) + Pod::Weaver::Section::Region (any)
 - (dev req) + Pod::Weaver::Section::Version (any)
 - (dev req) + Test::EOF (any)
 - (dev req) + Test::EOL (any)
 - (dev req) + Test::More 0.88
 - (dev req) + Test::NoTabs 1.4
 - (dev req) + Test::Version 1
 - (dev req) + Test::Warnings 0.026
 - (dev sug) + Dist::Iller 0.1408
 - (dev sug) + Dist::Iller::Config::Author::CSSON 0.0322
 - (dev sug) + Dist::Iller::Config::Author::CSSON::MapMetro 0.0322

0.1001 2015-02-03T22:23:14Z CET
 [Documentation]
 - Improved synopsis (command line example) and cleanup.

 [Requirements]
 - Bump Map::Metro to 0.2206

0.1000 2015-01-07T00:47:01Z CET
 - Initial release


lib/Map/Metro/Plugin/Map/Brussels.pm
use 5.14.0;
use strict;
use warnings;

package Map::Metro::Plugin::Map::Brussels;

# ABSTRACT: Map::Metro map for Brussels
# AUTHORITY
our $VERSION = '0.1003';

use Moose;
with 'Map::Metro::Plugin::Map';

has '+mapfile' => (
    default => 'map-brussels.metro',
);
sub map_version {
    return $VERSION;
}
sub map_package {
    return __PACKAGE__;
}

1;

__END__

=pod

=head1 SYNOPSIS

    use Map::Metro;
    my $graph = Map::Metro->new('Brussels')->parse;

Or:

    map-metro.pl route Brussels "Gare de l'Ouest" 'Centraal Station'

=head1 DESCRIPTION

See L<Map::Metro> for usage information.

=head1 Status

This map L<contains|Map::Metro::Plugin::Map::Brussels::Lines>:

=for :list
* The four metro lines I<L1>, I<L2>, I<L5> and I<L6> [L<wikipedia|https://en.wikipedia.org/wiki/Brussels_Metro>]
* The I<T3> tram line, but only between I<Gare du Nord/Noordstation> and I<Albert> [L<wikipedia|https://en.wikipedia.org/wiki/Brussels_tram_route_3>]

=for HTML <p><a href="https://raw.githubusercontent.com/Csson/p5-Map-Metro-Brussels/master/static/images/brussels.png"><img src="https://raw.githubusercontent.com/Csson/p5-Map-Metro-Brussels/master/static/images/brussels.png" style="max-width: 600px" /></a></p>

=cut


Makefile.PL
# This file was automatically generated by Dist::Zilla::Plugin::MakeMaker v6.012.
use strict;
use warnings;

use 5.016000;

use ExtUtils::MakeMaker;

use File::ShareDir::Install;
$File::ShareDir::Install::INCLUDE_DOTFILES = 1;
$File::ShareDir::Install::INCLUDE_DOTDIRS = 1;
install_share dist => "share";


my %WriteMakefileArgs = (
  "ABSTRACT" => "Map::Metro map for Brussels",
  "AUTHOR" => "Erik Carlsson <info\@code301.com>",
  "CONFIGURE_REQUIRES" => {
    "ExtUtils::MakeMaker" => 0,
    "File::ShareDir::Install" => "0.06"
  },
  "DISTNAME" => "Map-Metro-Plugin-Map-Brussels",
  "LICENSE" => "perl",
  "MIN_PERL_VERSION" => "5.016000",
  "NAME" => "Map::Metro::Plugin::Map::Brussels",
  "PREREQ_PM" => {
    "Map::Metro" => "0.2300",
    "Moose" => "2.1400",
    "namespace::autoclean" => "0.24"
  },
  "TEST_REQUIRES" => {
    "ExtUtils::MakeMaker" => 0,
    "File::Spec" => 0,
    "IO::Handle" => 0,
    "IPC::Open3" => 0,
    "Test::More" => "0.96"
  },
  "VERSION" => "0.1003",
  "test" => {
    "TESTS" => "t/*.t"
  }
);


my %FallbackPrereqs = (
  "ExtUtils::MakeMaker" => 0,
  "File::Spec" => 0,
  "IO::Handle" => 0,
  "IPC::Open3" => 0,
  "Map::Metro" => "0.2300",
  "Moose" => "2.1400",
  "Test::More" => "0.96",
  "namespace::autoclean" => "0.24"
);


unless ( eval { ExtUtils::MakeMaker->VERSION(6.63_03) } ) {
  delete $WriteMakefileArgs{TEST_REQUIRES};
  delete $WriteMakefileArgs{BUILD_REQUIRES};
  $WriteMakefileArgs{PREREQ_PM} = \%FallbackPrereqs;
}

delete $WriteMakefileArgs{CONFIGURE_REQUIRES}
  unless eval { ExtUtils::MakeMaker->VERSION(6.52) };

WriteMakefile(%WriteMakefileArgs);

{
package
MY;
use File::ShareDir::Install qw(postamble);
}
